lambda-buffers-compiler/src/LambdaBuffers/Compiler/ProtoCompat/Types.hs

```diff
@@ -68,6 +69,9 @@ data SourcePosition = SourcePosition {column :: Int, row :: Int}
   deriving stock (Show, Eq, Ord, Generic)
   deriving (Arbitrary) via GenericArbitrary SourcePosition
 
+defSourceInfo :: SourceInfo
+defSourceInfo = SourceInfo "" (SourcePosition 0 0) (SourcePosition 0 0)
+
```

Comment:
I know community is against the use of it - but in this case it could be useful to do it via the [library and Class](https://hackage.haskell.org/package/data-default-0.7.1.1/docs/Data-Default.html)

Reply:
I want to keep it ugly as this is a symptom that we should address proper. I'll add a todo above it.